src/components/cart/EmptyCart.jsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+
+export function EmptyCart() {
+  return (
+    <main className="empty-cart-main bg-white mx-auto pb-2">
+      <header>
+        <h1 className="h4 p-1 pb-sm">My Cart</h1>
+      </header>
+
+      <section
+        className="txt-center
+      "
+      >
+        <figure>
```

Comment:
Suggestion you could store the banner for empty cart on your assets folder in repo or host it elsewhere on CDN to be sure banner is always present

Reply:
Done